### ISSUE TITLE
splitting the dataset into the Training set and Test set into simple_linear_regression

### ISSUE THREAD

**Issue by ajallal88** [ajallal88/Machine-Learning#14]
Write a code to split the dataset into the Training set and Test set into simple_linear_regression.py file
